org.newerth.savagexr.json

```diff
@@ -0,0 +1,121 @@
+{
+	"app-id": "org.newerth.savagexr",
+	"branch": "master",
+	"runtime": "org.freedesktop.Platform", 
+	"sdk": "org.freedesktop.Sdk", 
+	"runtime-version": "1.6", 
+	"tags": ["proprietary"],
+	"command": "bootstrapsavagexr.sh",
+	"build-options": {
+		"cflags": "-O3 -pipe",
+		"cppflags": "-O3 -pipe"
+	},
+	"finish-args": [
+		"--socket=x11",
+		"--socket=wayland",
+		"--share=network",
+		"--share=ipc",
+		"--device=dri",
+		"--socket=pulseaudio"
+	],
+	"modules": [
+		{
+			"name": "wget",
+			"buildsystem": "autotools",
```

Comment:
you don't *need* this bit, as autotools is the default.

Reply:
removed.